src/storage-types/src/sources/sql_server.rs

```diff
@@ -41,12 +41,44 @@ pub const SNAPSHOT_MAX_LSN_WAIT: Config<Duration> = Config::new(
     CDC to be fully enabled) before taking an initial snapshot.",
 );
 
+pub const SNAPSHOT_PROGRESS_REPORT_INTERVAL: Config<Duration> = Config::new(
+    "sql_server_snapshot_progress_report_interval",
+    Duration::from_secs(2),
+    "Interval at which we'll report progress for currently running snapshots.",
+);
+
 pub const CDC_POLL_INTERVAL: Config<Duration> = Config::new(
     "sql_server_cdc_poll_interval",
     Duration::from_millis(500),
     "Interval at which we'll poll the upstream SQL Server instance to discover new changes.",
 );
 
+pub const CDC_CLEANUP_CHANGE_TABLE: Config<bool> = Config::new(
+    "sql_server_cdc_cleanup_change_table",
+    true,
+    "When enabled we'll notify SQL Server that it can cleanup the change tables \
+    as the source makes progress and commits data.",
+);
```

Comment:
when would we want to not do this?

Reply:
Should have added this as a comment, but this is a CYA flag just in case something goes wrong, e.g. we're too eagerly cleaning up data